### ISSUE TITLE
Task scheduler

### ISSUE THREAD

**Issue by Mrkkkkc** [Rem0o/FanControl.Releases#171]
Not entirely sure if it's something to do with the program or not, but setting a task in task scheduler for auto start the program doesn't work at all, only manual start up works

**Comment by Rem0o** [comment]
Did you try the start with windows checkbox in the program itself? It creates a task in the task scheduler with the proper parameters for it to work.

**Comment by Mrkkkkc** [comment]
yes, that does not work either

**Comment by Mrkkkkc** [comment]
Also in addition the application failed to load sensor data when launched
numerous times, though this is usually solved with exiting and re-opening

**Comment by Rem0o** [comment]
If it fails to detect sensors, it means something else is interfering ( software ) or something in not properly initialized yet, usually if your start it too soon after boot.

**Comment by emil2424** [comment]
I use my own BAT file to run all the programs I use for the GPU OC simultaneously.
`@echo off
start "Program 1"  "C:\Program Files (x86)\MSI Afterburner\MSIAfterburner.exe"
start "Program 2"  "C:\Program Files (x86)\RivaTuner Statistics Server\SDK\Tools\DesktopOverlayHost\Release\DesktopOverlayHost.exe"
start "Program 3"  "C:\Program Files (x86)\FanControl\FanControl.exe"
@echo on`

Unfortunately, FanControl fails to start. My programming skills are basically nonexistent :)
What parameters should I add to line Program 3

**Comment by Rem0o** [comment]
Are you running as admin? FanControl needs admin.

**Comment by emil2424** [comment]
Problem solved :)
Of course I was running FanControl with admin rights.
The log generated after trying to run the program showed references to the wrong directory.

In the code of the BAT file I defined the path C: \ Program Files (x86) \ FanControl \ FanControl.exe 
and the log was:
`An exception was thrown while activating λ: FanControl.Views.TrayIconView. ---> System.IO.FileNotFoundException: C: \ Users \ MyName \ Desktop \ FanControl.exe ............ `

I guess my BAT script despite defining the path, used the location of the BAT file (desktop) so I moved it to the FanControl directory and it completely solved the problem :)
